MODULE.bazel

```diff
@@ -1,3 +1,17 @@
+# Copyright 2024 The StableHLO Authors. All Rights Reserved.
```

Comment:
why did this not fail on the submission?

Reply:
Are you wondering about the license check we have for CI? `addlicense` doesn't support this: https://github.com/openxla/stablehlo/pull/1929

We should probably just drop `addlicense` and replace it with `find` + `grep`. I plan to send a PR for that later today.

Reply:
make sense Or can we add the *.bazel file extension to `addlicense `. With that any newly added bazel might should have the copyright ( note that  all the current bazel files in stablehlo/* now have the header)

Reply:
> Add the .bazel file extension to addlicense

I tried: https://github.com/google/addlicense/pull/154

> Why don't we fork addlicense

It would be easy to fork and patch addlicense, but I doubt we want the maintenance burden that could come with that. Let me look into a `find` + `grep` based license check, IMO that will be much less work now and in the future.